great_tables/_options.py

```diff
@@ -969,3 +969,673 @@ def opt_all_caps(
         res = tab_options(res, row_group_text_transform="inherit")
 
     return res
+
+
+def opt_stylize(self: GTSelf, style: int = 1, color: str = "blue") -> GTSelf:
+    """
+    Stylize your table with a colorful look.
+
+    With the `opt_stylize()` method you can quickly style your table with a carefully curated set of
+    background colors, line colors, and line styles. There are six styles to choose from and they
+    largely vary in the extent of coloring applied to different table locations. Some have table
+    borders applied, some apply darker colors to the table stub and summary sections, and, some even
+    have vertical lines. In addition to choosing a `style` preset, there are six `color` variations
+    that each use a range of five color tints. Each of the color tints have been fine-tuned to
+    maximize the contrast between text and its background. There are 36 combinations of `style` and
+    `color` to choose from.
+
+    Parameters
+    ----------
+    style : int
```

Comment:
We don't need to write annotations in the parameters section of a docstring. griffe + quartodoc will add the annotations from the function signature.